### DISCUSSION TITLE
Photoprism command not found - with solution

### DISCUSSION THREAD

**Discussion by WkdPaul** [tteck/Proxmox#3313] [General] [answered]
I installed Photoprism through the script and was up within minutes ! It's pretty great BUT ... when I tried to customize the install through the CLI, I got the error ;

photoprism: command not found

So I'll post this here in case someone gets the same issue but can't figure it out. Turns out the script is missing one line at the end so that the photoprism command works ;

sudo ln -sf /opt/photoprism/bin/photoprism /usr/local/bin/photoprism

If you're having the same issue, simply copy/paste the line above in your console so that the photoprism command starts working. After that I was able to add users through the CLI (it's not available in the webui unless you get the Pro version).

**Comment by tteck** [comment, marked as answer]
Added https://github.com/tteck/Proxmox/commit/5fcae037c20885d431959313c573057f0a488699

**Reply by WkdPaul** [reply]
> Added [5fcae03](https://github.com/tteck/Proxmox/commit/5fcae037c20885d431959313c573057f0a488699)

Amazing! Didn't think it would get a fix that quickly!!!
